learning-bitmark/contributing-to-bitmark/bitmark-software-development-process.md

```diff
@@ -0,0 +1,127 @@
+# Current development workflow
+
+**Right now we have these projects:**
+* bitmarkd
+* Core API
+* Updaterd
+* Backend services
+* Bitmark SDK
+
+**Development process:**
+
+Our current workflow uses OKR, where we define sprints as a deadline to deliver the software artefact, but we do not limit ourselves. We do translate the OKR in ISSUES on Github where we can track them. The external contributors can get involved with the project opening new ISSUES, being part of the discussion in existent ISSUES and make PULL REQUESTS within our projects.
+
+![](https://github.com/bitmark-inc/docs/blob/master/learning-bitmark/contributing-to-bitmark/imgs/project_bitmarkd.png)
+
+
+**Branch, Tag and release process:**
+
+We have a single development branch master, from where we create tags to perform minor and major releases. The development never happens directly on branch master, instead the developer create a development branch from master, after the developer believes its branch is ready, it opens a pull request and assign another developer to make a review. After the review process, the developer merges its branch to master branch and deletes the development branch. The developers never commit directly into the master branch without passing through the review process.
```

Comment:
I think I've suggested elsewhere using Bitmark instead of "we". It's easy to get tied up with a lot of "we" and "they" and "you" in docs, and this is an easy one to cut out.